.github/workflows/release.yml

```diff
@@ -30,7 +31,7 @@ jobs:
         run: antsibull-changelog release -v --version "${{ steps.version.outputs.next-version }}"
 
       - name: "Run antsichaut"
-        uses: rndmh3ro/antsichaut-action@main
+        uses: gardar/antsichaut-action@latest  # Until new pip release gets published:  https://github.com/rndmh3ro/antsichaut/issues/7
```

Comment:
Looks like the issue is fixed.

Reply:
Since the release is not working due to antsichaut action failing I dug a bit into what is happening. It seems that `gardar/antsichaut-action@latest` is using `pip` to install latest version of `antsichaut` (0.3.0 at the time of writing) and for some reason, this version is NOT containing the same code as in https://github.com/rndmh3ro/antsichaut repository. For example this [block of code](https://github.com/rndmh3ro/antsichaut/blob/master/antsichaut/antsichaut.py#L176-L179) looks completely different in pip installed version:
```python
         write_fragments = True
         if write_fragments:
             data = {}
         else:
             with open(
                 "changelogs/changelog.yaml",
             ) as file:
                 data = yaml.load(file)
```

Because of this issue, the github action fails, and thus release process breaks.

Issue tracking this is in https://github.com/rndmh3ro/antsichaut/issues/8

cc @gardar @SuperQ